### PR TITLE
Update simulator documentation

### PR DESCRIPTION
**Pull request by WillB97**
Largely reproduces https://docs.roboticsoutreach.org/docs/simulator/

Pairs with https://github.com/srobo/sbot_simulator and https://github.com/srobo/sr-robot3/pull/17

### CODE REVIEW

File: simulator/simulated_robot.md [hunk 1]
```diff
@@ -8,135 +8,97 @@ title: The Simulated Robot
...
+This is achieved by returning the relative red content of the surface directly below the sensor.
 
-<!-- Changing this? Consider updating the "Simulated robot inputs" section in `simulator/using_the_simulator.md` -->
+The measured values can then be read using the [Analog Input]({{ site.baseurl }}/programming/arduino/sr_firmware#input) interface.
+This is returned as a voltage between 0 and 5 volts, with lower values indicating a darker surface.
 
-The simulated robot has a camera which provides position and orientation
-information about other objects within the simulation. This simulates the
-system of fiducial markers which the physical robot's camera can detect.
+These appear as blue rectangles on the robot model.
```
Comment (Adimote) [inline simulator/simulated_robot.md:87]: [nit]
```suggestion
These appear as blue rectangles on the bottom of the robot model.
```

File: simulator/simulated_robot.md [hunk 2]
```diff
@@ -8,135 +8,97 @@ title: The Simulated Robot
...
+![Reflectance Sensors]({{ site.baseurl }}/images/content/simulator/robot-bottom.png)
 
-If you want to see for yourself what the robot camera can see, you can enable an
-overlay of the camera's view by right clicking on the robot then going to
-**Overlays** &rarr; **Camera Devices** &rarr; **Show 'camera' overlay**.
+### Microswitches
 
-![An image of the camera overlay with the relevant part of the simulated arena in the background]({{ site.baseurl }}/images/content/simulator/camera-overlay.png)
+On the front and back of the robot, there are microswitches that can detect if the robot has bumped into something.
+These appear as red cuboids on the robot model.
```
Comment (Adimote) [inline simulator/simulated_robot.md:94]: [nit]
```suggestion
These appear as red cuboids on the front and back of the robot model.
```

**Review by Adimote — APPROVED** [review]

File: simulator/setting_up_simulator.md [hunk 3]
```diff
@@ -6,118 +6,76 @@ title: Setting up the Simulator
...
-    └── robot.py
-    ```
-    If there is not an existing `robot.py` an example one will be created when the simulator first runs.
+First, navigate to the folder you extracted the simulator into. This folder should contain a file called `setup.py`.
+Run this script and it will set up the environment for you.
+A terminal window will open and you will see the output of the script, if there are any errors displayed you should ask for help in [Discord][discord].
 
-4. Open the Webots IDE, then use that to open the `worlds/Arena.wbt` file.
+<div class="info" markdown="1">
+In order to run the Python script, instead of opening the file you may need to right-click and select **Open with** &rarr; **Python**.
```
Comment (Adimote) [inline simulator/setting_up_simulator.md:58]: What's the mac equivalent to "Right click > Open With..."?

File: programming/arduino/custom_firmware.md [hunk 4]
```diff
@@ -12,11 +12,6 @@ On this page we talk specifically about Arduinos.
 However this page is applicable to opening any device that shows up as a serial port.
 </div>
 
-<div class="info">
```
[the remark this answers is not in the document]
Comment (WillB97) [reply programming/arduino/custom_firmware.md:15]: No, see https://github.com/srobo/docs/blob/05f55cdfc25549b03f3f652f30144f81800b41b8/simulator/simulated_robot.md?plain=1#L33

File: simulator/simulated_robot.md (hunk 2 above)
Comment (WillB97) [reply simulator/simulated_robot.md:94]: The location has already been covered in the sentence before. Adding it here breaks the flow of the paragraph.

File: simulator/setting_up_simulator.md (hunk 3 above)
Comment (WillB97) [reply simulator/setting_up_simulator.md:58]: The same.

**Pull request merged**